pkgs/top-level/ocaml-packages.nix

```diff
@@ -412,6 +412,11 @@ let
       then callPackage ../development/ocaml-modules/ppx_deriving {}
       else null;
 
+    ppx_import =
+      if lib.versionAtLeast ocaml.version "4.02"
```

Comment:
Instead of this `if`-clause, rather use `minimumSupportedOcamlVersion` in the derivation itself. (We probably should refactor all `ocamlPackages` to use this style)